lesson16/js/index.js

```diff
@@ -1,5 +1,173 @@
 "use strict";
 
+// Task 1
+
+let x = 20;
+let y = 58;
+let z = 42;
+
+console.log(x + y + z);
+
+// Task 2
+
+let userAge = +prompt('Enter your age: ', '');
+let secondsInMinute = 60;
+let minuteInHour = 60;
+let houresInDay = 24;
+let daysInYear = 365;
```

Comment:
В этих случаях предпочтительнее const
```suggestion
const SECONDS_IN_MINUTE = 60;
const MINUTES_IN_HOUR = 60;
const HOURS_IN_DAY = 24;
const DAYS_IN_YEAR = 365;
```

Большие буквы, так как это факты, которые "никогда" не поменяются